site/content/docs/user/quick-start.md

```diff
@@ -344,6 +344,27 @@ nodes:
 - role: worker
 ```
 
+#### Limit node resources
+`kind` implements [docker resource constraints][docker resource constraints],
```

Comment:
We should not implement docker anything. We just got ~done putting that behind some interfaces :+)
We'll want to do podman etc.

Reply:
We can explain limits without this